### DISCUSSION TITLE
How to turn off schema validation for a particular resolver?

### DISCUSSION THREAD

**Discussion by cjnoname** [MichalLytek/type-graphql#1454] [Q&A]
How to turn off schema validation for a particular resolver?

**Comment by MichalLytek** [comment]
You mean input/args validation?

**Reply by cjnoname** [reply]
Yes or no mate, now I am facing an issue that I renamed EnumA to EnumB on an resolver, and some of the mobile app users haven't got a chance to upgrade the app, then there's show an validation error:

{
  "error": {
    "errors": [
      {
        "message": "Variable \"$aaa\" of type \"EnumA!\" used in position expecting type \"EnumB!\".",
        "extensions": {
          "code": "GRAPHQL_VALIDATION_FAILED"
        }
      }
    ]
  }
}

**Reply by cjnoname** [reply]
Actually A and B are same Enum with different names, now I am facing the issue that I cannot simply rename the enum name, otherwise the other group of users won't be albe to use

**Reply by MichalLytek** [reply]
You can't disable runtime validation. You need to adjust your schema in a way that it still support old apprach and the new one.
It's not TypeGraphQL limitation. It's just GraphQL spec rules, you can't run any random query and expect reasonable results being returned.
